app-office/calcurse/calcurse-4.3.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A calendar and scheduling application for the command line"
+DESCRIPTION="calcurse is a calendar and scheduling application for the \
+command line. It helps keep track of events, appointments and everyday tasks. \
+A configurable notification system reminds user of upcoming deadlines, the \
+curses based interface can be customized to suit user needs and a very \
+powerful set of command line options can be used to filter and format \
+appointments, making it suitable for use in scripts."
+HOMEPAGE="http://calcurse.org/"
+COPYRIGHT="calcurse Development Team"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://calcurse.org/files/calcurse-$portVersion.tar.gz"
+CHECKSUM_SHA256="31ecc3dc09e1e561502b4c94f965ed6b167c03e9418438c4a7ad5bad2c785f9a"
+PATCHES="calcurse-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]
+then
+	commandBinDir=$binDir
+	commandSuffix=$secondaryArchSuffix
+else
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	calcurse$secondaryArchSuffix = $portVersion
+	cmd:calcurse$commandSuffix
+	cmd:calcurse_caldav$commandSuffix
+	cmd:calcurse_upgrade$commandSuffix
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libncursesw$secondaryArchSuffix
+"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autopoint
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+"
+
+BUILD()
+{
+	autoreconf -vfi
+	# define _BSD_SOURCE for /boot/system/develop/headers/bsd/paths.h
+	CFLAGS="-D_BSD_SOURCE" runConfigure --omit-dirs binDir ./configure \
+		--bindir=$commandBinDir
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+}
```

Comment:
I've noticed that the project's Makefile has a test suite which we can use.
So you can add a TEST function after INSTALL (with a blank line between their blocs.)
So this is what you can add:
```

TEST()
{
	make check
}
```